Formula/clickhouse-odbc.rb

```diff
@@ -0,0 +1,48 @@
+class ClickhouseOdbc < Formula
+  desc "ClickHouse ODBC driver"
+  homepage "https://clickhouse.yandex"
+  url "https://github.com/yandex/clickhouse-odbc.git",
+    :tag => "v1.0.0.20180903",
+    :revision => "17ecca3993e164d2d49360ed9be68becc11b7b05"
+  # head "https://github.com/yandex/clickhouse-odbc.git"
```

Comment:
No head

Reply:
brew audit --strict says me remove head and other options below. i think all of them useful and should not be commented.